defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+smartd_devices:
```

Comment:
Where are my three dashes?

Reply:
I would've bet you'd think of these as optional bloat. I'll be sure to add them ;)